### PR TITLE
[ePD] Validations to be confirmed or developed on partner or unicef accept

### PR DESCRIPTION
**Pull request by roman-karpovich**

### CODE REVIEW

File: src/etools/applications/partners/validation/interventions.py [hunk 1]
```diff
@@ -319,6 +319,14 @@ def cp_structure_valid(i):
     return True
 
 
+def pd_outputs_presented(i):
```
Comment (robertavram) [inline src/etools/applications/partners/validation/interventions.py:322]: at least one pd output is present `pd_outputs_present`

File: src/etools/applications/partners/validation/interventions.py [hunk 2]
```diff
@@ -319,6 +319,14 @@ def cp_structure_valid(i):
...
+    return i.result_links.exists()
+
+
+def pd_outputs_are_linked_to_indicators(i):
+    return not i.result_links.filter(ram_indicators__isnull=True).exists()
```
Comment (robertavram) [inline src/etools/applications/partners/validation/interventions.py:327]: this is supposed to check for PD indicators which are the `AppliedIndicator` model, something like:
`i.result_links.filter(ll_results__applied_indicators__isnull=True)`

**Review by robertavram — CHANGES_REQUESTED** [review]

File: src/etools/applications/partners/validation/interventions.py (hunk 1 above)
Comment (roman-karpovich) [reply src/etools/applications/partners/validation/interventions.py]: fixed

File: src/etools/applications/partners/permission_matrix/intervention_permissions.csv [hunk 3]
```diff
@@ -133,7 +136,7 @@ Field no,Field Name,Group,Condition,Status,Action,Allowed
 3.13.1,reporting_requirements,Unicef Focal Point,termination_doc_attached,Active,Edit,TRUE
 3.13.1,reporting_requirements,Unicef Focal Point,termination_doc_attached,Signed,Edit,TRUE
 3.13.1,reporting_requirements,UNICEF User,prp_server_on,*,View,TRUE
-3.13.1,reporting_requirements,*,unicef_not_accepted,Draft,Required,FALSE
```
[the remark this answers is not in the document]
Comment (roman-karpovich) [reply src/etools/applications/partners/permission_matrix/intervention_permissions.csv:136]: configured test to confirm partner has no view permission to this field

File: src/etools/applications/partners/validation/interventions.py (hunk 2 above)
Comment (roman-karpovich) [reply src/etools/applications/partners/validation/interventions.py]: updated

**Pull request merged**